pkpdapp/pkpdapp/templates/project_detail.html

```diff
@@ -6,27 +6,38 @@
 
 {% extends "base/base.html" %}
 {% load static %}
+{% load markdownify %}
 
 {% block main_content %}
+
 <p></p>
-<h2>Project {{project.name}}</h2>
-<p>{{project.description}}</p>
+<h2>
+  Project {{project.name}}
+</h2>
+(<a href="{% url 'project-update' project.id %}"> Update </a>)
```

Comment:
"Update project information"?

Reply:
I know it's clear to us what it means to update a project but a user doesn't know about the project models...

Reply:
yea, that sounds better